frappe/public/js/frappe/form/controls/time.js

```diff
@@ -43,6 +43,9 @@ frappe.ui.form.ControlTime = class ControlTime extends frappe.ui.form.ControlDat
 		};
 	}
 	set_input(value) {
+		if (!value) {
+			value = frappe.datetime.now_time(true)
```

Comment:
This is also probably invalid. 

User can choose not to set any value or erase existing value from field, the error was only about filters where empty value is problematic.